lib/matplotlib/patches.py

```diff
@@ -1469,32 +1469,30 @@ class Annulus(Patch):
 
     def __str__(self):
         if self.a == self.b:
-            fmt = "Annulus(xy=(%s, %s), r=%s, width=%s, angle=%s)"
-            pars = (self.center[0], self.center[1],
-                    self.a, self.width, self.angle)
+            r = str(self.a)
         else:
-            fmt = "Annulus(xy=(%s, %s), a=%s, b=%s, width=%s, angle=%s)"
-            pars = (self.center[0], self.center[1],
-                    self.a, self.b, self.width, self.angle)
-        return fmt % pars
+            r = '(%s, %s)' % (self.a, self.b)
+
+        return "Annulus(xy=(%s, %s), r=%s, width=%s, angle=%s)" % \
```

Comment:
Should be `%r`, not `%s`, since this is within a repr.

Reply:
I think it's probably OK since it's inside a __str__ no?